src/ui/app/jsx/repair-list.jsx

```diff
@@ -271,22 +270,38 @@ const repairList = React.createClass({
       obs.subscribeOnNext(names => this.setState({clusterNames: names}))
     );
 
-    this._repairsSubscription = this.props.repairs.subscribeOnNext(obs =>
+// @TODO: Remove this. The id is a UUID, so I have no idea how this was working. Need to check with Mick.
+//    this._repairsSubscription = this.props.repairs.subscribeOnNext(obs =>
+//      obs.subscribeOnNext(repairs => {
+//        const sortedRepairs = Array.from(repairs);
+//        sortedRepairs.sort((a, b) => a.id - b.id);
+//        this.setState({repairs: sortedRepairs});
+//      })
+//    );
+
```

Comment:
i say remove this^ commented section! sorting of the repairs in done below at https://github.com/thelastpickle/cassandra-reaper/pull/662/files#diff-82c4a2e9d75b730d5b84d59d8d0b5227R375 and https://github.com/thelastpickle/cassandra-reaper/pull/662/files#diff-82c4a2e9d75b730d5b84d59d8d0b5227R388

Reply:
Agreed.